pkg/tasks/clustermonitoringoperator.go

```diff
@@ -54,6 +54,26 @@ func (t *ClusterMonitoringOperatorTask) Run() error {
 		return errors.Wrap(err, "reconciling cluster-monitoring ClusterRole failed")
 	}
 
+	cr, err = t.factory.ClusterMonitoringRulesEditClusterRole()
```

Comment:
This is the edit cluster rule not that we are editing the cluster role, correct? :)

Reply:
correct